src/js/grades/rangeDefs.ts

```diff
@@ -70,3 +70,67 @@ export const YDS_DEFS: SliderMarksType =
     score: getScoreForSort('5.15', GradeScales.Yds)
   }
 }
+
+export const BOULDER_DEFS: SliderMarksType =
+{
+  0: {
+    score: -1,
+    label: 'V-easy'
+  },
+  1: {
+    score: getScoreForSort('v0', GradeScales.VScale),
+    label: 'V0'
+  },
+  2: {
+    score: getScoreForSort('v1', GradeScales.VScale),
+    label: 'V1'
+  },
+  3: {
+    label: 'V2',
+    score: getScoreForSort('v2', GradeScales.VScale)
+  },
+  4: {
+    label: 'V3',
+    score: getScoreForSort('v3', GradeScales.VScale)
+  },
+  5: {
+    label: 'V4',
+    score: getScoreForSort('v4', GradeScales.VScale)
+  },
+  6: {
+    label: 'V5',
+    score: getScoreForSort('v5', GradeScales.VScale)
+  },
+  7: {
+    label: 'V6',
+    score: getScoreForSort('v6', GradeScales.VScale)
+  },
+  8: {
+    label: 'V7',
+    score: getScoreForSort('v7', GradeScales.VScale)
+  },
+  9: {
+    label: 'V8',
+    score: getScoreForSort('v8', GradeScales.VScale)
+  },
+  10: {
+    label: 'V9',
+    score: getScoreForSort('v9', GradeScales.VScale)
+  },
+  11: {
+    label: 'V10',
+    score: getScoreForSort('v10', GradeScales.VScale)
+  },
+  12: {
+    label: 'V11',
+    score: getScoreForSort('V11', GradeScales.VScale)
+  },
+  13: {
+    label: 'V12',
+    score: getScoreForSort('v12', GradeScales.VScale)
+  },
+  14: {
```

Comment:
This is a part I am unsure about. I don't think will capture everything above v13+, but rather up to V13+. I am not sure how to account for this? Perhaps increase number of tick marks to account for v grades up to what is currently the max. A quick google suggests the highest grade out there is a v17 (https://www.99boulders.com/bouldering-grades)